dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/fixturegenerator/FrameworkFixtureGeneratorBuilder.kt

```diff
@@ -81,7 +81,12 @@ class FrameworkFixtureGeneratorBuilder(
 
     private fun buildDataFixtures(dataFixturesTsPath: Path) {
         val imports = rootSectionBuilder.imports +
-            TypeScriptImport("generateFixtureDataset", "@e2e/fixtures/FixtureUtils")
+            TypeScriptImport("generateFixtureDataset", "@e2e/fixtures/FixtureUtils") +
+            TypeScriptImport(
```

Comment:
Is the imported api model actually used? I only went through the files for the older integrated frameworks and there only the input was added. I'll see if so. You can ignore this. For now.

Reply:
Ja es wird benutzt.  Es soll nur zsm mit den anderen Sachen aus "clients/backend" importiert werden.